application/backend/app/alembic/versions/da385d690aae_schema.py

```diff
@@ -151,5 +164,6 @@ def downgrade() -> None:
     op.drop_table("dataset_revisions")
     op.drop_table("sources")
     op.drop_table("sinks")
+    op.drop_table("training_configurations")
```

Comment:
Move to L160 (begin of list), that's where Alembic would put this line if this code was auto-generated (I assume you edited it manually).